proposals/0071-Entity/01-context.md

```diff
@@ -0,0 +1,455 @@
+
+**Direct Scraping**
+
+In direct scraping, Prometheus discovers and scrapes each target individually. Service Discovery provides accurate metadata about each target, because the target *is* the entity producing metrics.
+
+```
+┌─────────────┐
+│  Service A  │◀────┐
+│  (pod-xyz)  │     │
+└─────────────┘     │
+                    │ scrape    ┌───────────┐
+┌─────────────┐     ├──────────▶│           │
+│  Service B  │◀────┤           │Prometheus │
+│  (pod-abc)  │     │           │           │
+└─────────────┘     │           └───────────┘
+                    │
+┌─────────────┐     │
+│  Service C  │◀────┘
+│  (pod-def)  │
+└─────────────┘
+```
+
+Here, Kubernetes SD knows that `pod-xyz` runs Service A with specific labels, resource limits, and node placement. This metadata accurately describes the entity producing metrics—SD-derived entities work well.
+
+**Gateway and Federation**
+
+In gateway architectures, metrics flow through an intermediary before reaching Prometheus. The intermediary aggregates metrics from multiple sources.
+
+```
+┌───────────┐     ┌───────────┐       ┌───────────┐
+│ Service A │────▶│           │       │           │
+│           │push │   OTel    │──────▶│Prometheus │
+├───────────┤     │ Collector │scrape │           │
+│ Service B │────▶│           │       │           │
+│           │     │(gateway)  │       │           │
+├───────────┤     │           │       │           │
+│ Service C │────▶│           │       │           │
+└───────────┘     └───────────┘       └───────────┘
+```
+
+Here, SD only sees the OTel Collector—not Services A, B, or C. Any SD-derived metadata would describe the collector, not the actual metric producers. The same applies to Prometheus federation and pushgateway patterns.
+
+| What SD Sees | What Actually Produced Telemetry |
+|--------------|----------------------------------|
+| `otel-collector-pod-xyz` | `payment-service`, `auth-service`, `user-service` |
+| `prometheus-federation-1` | Hundreds of scraped targets from regional Prometheus |
+| `pushgateway-xyz` | Various batch jobs and short-lived processes |
+| `kube-state-metrics-0` | Workloads running in K8s and K8s API itself |
+
+**Supporting Both Models**
+
+This proposal must support both architectures:
+
+1. **Direct scraping**: Entity information can be derived from Service Discovery metadata, since SD accurately describes each target.
+2. **Gateway/federation**: Entity information must be embedded in the exposition format to travel with the metrics through intermediaries.
+
+Users choose the appropriate approach for their architecture. See [Service Discovery](./04-service-discovery.md) for configuration details.
+
+---
+
+## Goals
+
+This proposal aims to achieve the following:
+
+### 1. Define Entity as a Native Concept
+
+Prometheus should recognize Entities as a distinct concept with their own semantics, separate from metrics. Entities represent the things that produce telemetry, not the telemetry itself.
```

Comment:
How is this a better approach than the existing idea of modeling Resources and Entities as [persisted Prometheus metadata](https://docs.google.com/document/d/1epBslSSwRO2do4armx40fruStJy_PS6thROnPeDifz8/edit?tab=t.0#heading=h.5sybau7waq2q)?